blueocean-commons/src/main/java/io/jenkins/blueocean/commons/DigestUtils.java

```diff
@@ -10,6 +11,38 @@
 public class DigestUtils
 {
 
+    public static String sha256(byte[] original) {
+        try {
+            MessageDigest digest = MessageDigest.getInstance("SHA-256");
+            byte[] encodedhash = digest.digest(original);
+            StringBuilder sb = new StringBuilder(2+encodedhash.length);
+            for(byte b:encodedhash){
+                sb.append(b);
+            }
+            return sb.toString();
+        } catch (NoSuchAlgorithmException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+    }
+
+    public static String sha256(String original) {
+        return sha256(original, StandardCharsets.UTF_8);
+    }
+
+    public static String sha256(String original, Charset charset) {
+        try {
+            MessageDigest digest = MessageDigest.getInstance("SHA-256");
+            byte[] encodedhash = digest.digest(original.getBytes(charset));
+            StringBuilder sb = new StringBuilder(2+encodedhash.length);
+            for(byte b:encodedhash){
+                sb.append(b);
+            }
+            return sb.toString();
+        } catch (NoSuchAlgorithmException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
```

Comment:
Looks like this could be shortened to 
```suggestion
        sha256(original.getBytes(charset));
```